apps/web/src/pages/integrations/IntegrationsListModal.tsx

```diff
@@ -161,10 +161,11 @@ export function IntegrationsListModal({
       opened={isOpen}
       onClose={handleModalClose}
       styles={{ modal: { overflowX: 'hidden', overflowY: sidebarType ? 'hidden' : 'auto' } }}
+      data-test-id="integrations-list-modal"
     >
       <IntegrationsList
         withOutlet={false}
-        onAddProviderClick={showSelectSidebar}
+        onAddProviderClick={() => showSelectSidebar()}
```

Comment:
fixed a small bug that was passing event instead of value, but in this case we don't have to pass anything